src/components/view/ListView.vue

```diff
@@ -141,11 +144,14 @@
       <router-link :to="{ path: '/pod/' + record.podid }">{{ text }}</router-link>
     </a>
     <a slot="account" slot-scope="text, record" href="javascript:;">
-      <router-link :to="{ path: '/account/' + record.accountid }" v-if="record.accountid">{{ text }}</router-link>
+      <router-link
+        :to="{ path: `${$route.path}/${record.account}`, query: { account: record.account, domainid: record.domainid, quota: true } }"
+        v-if="record.accountid && 'quota' in record">{{ text }}</router-link>
```

Comment:
Instead the check must be based on $route.name or path

Reply:
I've changed it to check $route.path